README.md

```diff
@@ -171,7 +171,7 @@ make graph-properties-convert
 Many Galois/Lonestar applications work with graphs. We store graphs in a binary format
```

Comment:
Galois/Lonestar --> katana/lonestar. 
The wording in this file needs to be changed a bit graph-convert does provide '-gr2kg' now but that can be done in a separate PR.